components/service/nimbus/src/main/java/mozilla/components/service/nimbus/Nimbus.kt

```diff
@@ -160,7 +167,8 @@ object Nimbus : NimbusApi {
 
     override fun optOutAll() {
         if (!isInitialized) return
-        nimbus.optOutAll()
+        //nimbus.optOutAll()
+        throw RuntimeException("global opt-out hasn't landed in nimbus yet")
```

Comment:
Do we want to wait for this `optOutAll` API to land before cutting the sequence of releases?

Reply:
After speaking with @jhugman, I think yes, let's land that first.